src/dlsnode/util/aio/internal/JobQueue.d

```diff
@@ -129,6 +115,68 @@ public static struct Job
     ****************************************************************/
 
     private bool is_slot_free;
+
+    /****************************************************************
+
+        Buffer for the thread to fill and to copy to the user buffer
+        from.
+
+    ****************************************************************/
+
+    public void[] recv_buffer;
+
+    /***************************************************************
+
+        User buffer to copy the results to. NOTE: the reason to have
+        recv_buffer and user_buffer separate is because it might happen
+        that user discards the job and the buffer, while thread is
+        writting into it.
```

Comment:
Typo "writing".

Also add: "it might happen that user discards the job and the buffer, while thread is writing into it, in which case the contents of `rcv_buffer` are discarded."